packages/react-admin/src/sideEffect/saga/auth.js

```diff
@@ -33,7 +36,11 @@ export default authClient => {
                         type: USER_LOGIN_SUCCESS,
                         payload: authPayload,
                     });
-                    yield put(push(meta.pathName || '/'));
+                    let redirectTo = meta.pathName;
```

Comment:
Please use a ternary expression instead of a `let`

Reply:
Not sure what you mean here, a ternary expression using `yield` in the expression?  Can you show what you mean?

Reply:
```js
const redirectTo = meta.pathName || yield select(nextPathnameSelector);
```
?

Reply:
Parser says no. 
```
Unexpected token (39:54)
> 39 |                     const redirectTo = meta.pathName || yield select(nextPathnameSelector);
     |                                                       ^
  40 |                     yield put(push(redirectTo || '/'));
```

Reply:
Sorry:

```js
const redirectTo = yield meta.pathName || select(nextPathnameSelector);
```

Reply:
👍 **stupid** of me, I didn't realize you can yield any value.